src/modal/index.tsx

```diff
@@ -13,9 +13,9 @@ import style from './style'
 import KEYCODES from '_utils/keycodes'
 
 export interface ModalProps {
-  readonly close: () => void
+  readonly onClose: () => void
   readonly isOpen?: boolean
-  readonly children?: JSX.Element | JSX.Element[]
+  readonly children?: React.ReactNode
```

Comment:
I changed it to this, otherwise we would have to specify: string | number | JSX.Element, etc...